judo-requirement-report-test/src/test/java/hu/blackbelt/judo/requirement/test/TestRequirementProcessor.java

```diff
@@ -0,0 +1,238 @@
+     */
+    public void testReal(){
+        // Create Reference Array
+        List<String[]> refTable = createRefTable();
+
+        // Read generated csv
+        List<String[]> table = readCsv(System.getProperty("reportPath"));
+
+        // Check the lists
+        assertArrayEquals(refTable.toArray(), table.toArray());
+    }
+
+    private List<String[]> createRefTable() {
+        List<String[]> refTable = new LinkedList<>();
+        refTable.add(
+                new String[]{
+                        "TEST METHOD",
+                        "STATUS",
+                        "REQUIREMENT"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test05",
+                        "Missing annotation: @Test.",
+                        "R03"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test03",
+                        "OK",
+                        "R03"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test03",
+                        "OK",
+                        "R01"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test00",
+                        "There isn't any requirement id.",
+                        ""
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test04",
+                        "OK",
+                        "R04"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test02",
+                        "OK",
+                        "R02"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test01",
+                        "OK",
+                        "R01"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test04",
+                        "OK",
+                        "R05"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test05",
+                        "Missing annotation: @Test.",
+                        "R01"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test04",
+                        "OK",
+                        "R01"
+                }
+        );
+//        refTable.add(
+//                new String[]{
+//                        "",
+//                        "",
+//                        ""
+//                }
+//        );
```

Comment:
If commented code is no longer needed, please remove it.

Reply:
I left here because when somebody has to add a new element to the table, it can be faster and safer. She/he won't forget to change the values after the copying. :-) Therefore, I am leaving here, if it isn't brake any conde convesions of the team.

@robertcsakany What is your opinion?

Reply:
But there is a lot of example how can it be extended. I don't think it is neccesary.

Reply:
OK, I am removing it.